packages/data/lib/src/di/local_module.dart

```diff
@@ -17,11 +18,13 @@ Future<void> initLocalModule() async {
   final database = await $FloorMovieDatabase
       .databaseBuilder('app_database.db')
       .addMigrations([Migrations.migration1to2]).build();
+  final packageInfo = await PackageInfo.fromPlatform();
   inject.registerSingleton(instance: database);
   inject.registerFactory<MovieDao>(() => database.movieDao);
   inject.registerFactory<PeopleDao>(() => database.peopleDao);
   inject.registerFactory<DateDao>(() => database.dateDao);
   inject.registerSingleton(instance: await SharedPreferences.getInstance());
+  inject.registerFactory<PackageInfo>(() => packageInfo);
```

Comment:
why not singleton?

Reply:
in this case it redundant on my opinion, cuz version check runs only once, while running app, and no needed in future. So let it be a little less memory usage =)

Reply:
Okay